.github/workflows/ci-rust.yml

```diff
@@ -77,12 +77,12 @@ jobs:
         run: |
           set -ex
           AGNOSTIC_CRATES=`python misc/libparsec_crates_flags.py agnostic`
-          cargo nextest run --cargo-profile ci-rust $AGNOSTIC_CRATES
+          cargo nextest run --locked --cargo-profile ci-rust $AGNOSTIC_CRATES
```

Comment:
Instead of repeating the flags on each command, and eventually forget them if new commands are added in the future, they could be defined as `env` variable above. Something like:
```
CARGO_CI_FLAGS:--locked --cargo-profile ci-rust
```

Reply:
done.